core.js

```diff
@@ -1,8 +1,6 @@
 module.exports = {
 	env: {
-		'shared-node-browser': true,
```

Comment:
Okay. I think `mongo`, `mocha`, etc. should all be in their own env files, but I agree this one should be there.